.golangci.yml

```diff
@@ -23,7 +23,6 @@ linters-settings:
     # Correct spellings using locale preferences for US or UK.
     # Default is to use a neutral variety of English.
     # Setting locale to US will correct the British spelling of 'colour' to 'color'.
-    locale: UK
```

Comment:
Hope that is okay. Having to rewrite every comment, changing words to end in "ise", was slowly killing me. :)